CarpetX/interface.ccl

```diff
@@ -14,6 +14,7 @@ USES INCLUDE HEADER: vect.hxx
 USES INCLUDE HEADER: loop.hxx
 USES INCLUDE HEADER: loop_device.hxx
 
+INCLUDES HEADER: driver.hxx IN driver.hxx
```

Comment:
This change should be in a commit `CarpetX:`  and not `Loop`.

Reply:
You cannot include `CarpetX` files from `Loop` because `Carpetx` depends on `Loop`, and this would create a circular dependency.

Reply:
huh. I am speaking out of turn, I guess, yet: Uh, if Loop does not use the functionality from `driver.hxx` the I think I should not say `uses include header: driver.hxx`. By the same reasoning if the client thorns (say `NewRadX` ) directly use the information then _they_ should contain the `uses include header: driver.hxx` statement.

Reply:
My take is that circularity is an issue for the `REQUIRE:` statements (they affect compilation order so circles are not allowed) but not the `USES INCLUDE HEADER` statements I would say (handled before dependency is even known to Cactus).  
  
I agree with Erik's concern wrt CarpetX / Loop depending on each other in https://github.com/EinsteinToolkit/CarpetX/pull/300#discussion_r1821138682  (since we'd want a `REQUIRE` to go along with the `uses include header`).

Reply:
Do we _need_ this include in Loop? Since if there is the include I'd normally add a `REQUIRE` (otherwise Cactus may silently give you an empty include file). But the `REQUIRE` will introduce a circular dependency between `Loop` and `CarpetX`

Reply:
Well, what I'm proposing here would _not_ need the include (and hence the `REQUIRE`) in `Loop`. By itself, the `loop_outermost_int` routine does not use any information from the driver. On the other hand, the client thorns that intend to take the symmetry information from the driver and pass it to `loop_outermost_int`, _do_ need the include.

Reply:
The change still shows up in files changed as adding `INCLUDE HEADER "..."`. So if it is not used by Loop I'd suggest to remove the change.

Reply:
Hmm, from what I'm seeing the `USES INCLUDE HEADER` is only in `TestLoopX`, but not `Loop`. Can you point me to where it is?

Reply:
oha, sorry yes you are correct. I was looking at the very wrong file. In fact not even had I confused the thorn but also `INCLUDE HEADER` and `USES INCLUDE HEADER`.